components/02-molecules/alert/_yds-alert.scss

```diff
@@ -24,18 +26,31 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
   }
 
   &[data-alert-state='dismissed'] {
-    display: none;
+    @include tokens.animate(opacity, $alert-fade-speed);
```

Comment:
This seems to work, but I'm curious why you added it to the dismissed state, rather than to the `.alert` directly

Reply:
Oh! I think this isn't actually doing anything, because you're doing the animations to the "inner" element. See below about that.